selenium-log4j/src/main/java/io.sterodium.extensions/logging/GridStarter.java

```diff
@@ -0,0 +1,16 @@
+package io.sterodium.extensions.logging;
+
+import org.openqa.grid.selenium.GridLauncher;
+
+
+public final class GridStarter {
+
+    private GridStarter() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        Logging logging = new Logging(args);
+        logging.configure();
```

Comment:
Wrapper is needed to:
1. Configure logging in code.
2. Modify command line arguments that are passed to stock GridLauncher.